.github/workflows/quickstart_opentelemetry_ci.yml

```diff
@@ -0,0 +1,16 @@
+
+name: WildFly OpenTelemetry Quickstart CI
```

Comment:
Just to follow everyone else, please use opentelemetry-tracing instead of OpenTelemetry

Reply:
Changed.